meson.build

```
@@ -0,0 +1,111 @@
+project('MyPaint', 'cpp',
+  version : 'v2.1.0',
+  default_options : ['warning_level=3', 'cpp_std=c++14'])
+
+# Build machine info
+system_os = system()
```

No such function, you probably meant:
```suggestion
system_os = host_machine.system()
```